cves/CVE-2006-5752.yml

```diff
@@ -77,14 +87,14 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
+  answer: no
```

Comment:
Suggest adding an actual answer here and changing "no" to "false"

Reply:
added answer.